private/rdf4cpp/rdf/parser/IStreamQuadIteratorSerdImpl.cpp

```diff
@@ -6,7 +6,7 @@
 namespace rdf4cpp::rdf::parser {
 
 std::string_view IStreamQuadIterator::Impl::node_into_string_view(SerdNode const *node) noexcept {
-    return std::string_view{reinterpret_cast<char const *>(node->buf), static_cast<size_t>(node->n_bytes)};
+    return std::string_view{reinterpret_cast<char const *>(node->buf), node->n_bytes};
```

Comment:
```suggestion
    return std::string_view{static_cast<char const *>(node->buf), node->n_bytes};
```

`node->buf` is `void const *`, right?

Reply:
No its actually `uint8_t`

Reply:
oh. ok, then never mind.